.github/workflows/pipeline-deploy-app-develop.yaml

```diff
@@ -74,8 +74,8 @@ jobs:
       uses: ./.github/workflows/action-infrastructure-stack.yaml
       with:
         github_environment: dev
-        stack: capacity_management
-        action: apply
+        stack:
+        action: plan
```

Comment:
Happy with this ; the alternative could be to leave it as apply but change stack to github-runner ; that would at least keep the github runner stack up to date (eg if we change permissions).
Also happy to leave the code under capacity_management if we think that it may be useful later - but  will it ? Would we better to remove the stack altogether ? If we are keeping it could we rename it to application - to be consistent across the domain repos ?